src/widgets/OpenAI/OpenAI.jsx

```diff
@@ -82,10 +107,13 @@ export default function OpenAI({ functions, editor }) {
   };
 
   const handleSend = async () => {
-    if (data.current.request) {
+    if (data.current.request || transcript) {
```

Comment:
There is a small typo, it should be "transcribe"

Reply:
transcript is the variable that the useSpeechRecognition hook saves after converting the audio received from the user's microphone to text